tensorbay/cli/branch.py

```diff
@@ -25,7 +25,28 @@ def _implement_branch(obj: Dict[str, str], tbrn: str, verbose: bool) -> None:
     gas = get_gas(**obj)
     dataset_client = get_dataset_client(gas, info)
 
-    _list_branches(dataset_client, verbose)
+    if name:
+        _create_branch(dataset_client, name)
+    else:
+        _list_branches(dataset_client, verbose)
+
+
+def _create_branch(dataset_client: DatasetClientType, name: str) -> None:
+    if dataset_client.status.is_draft:
+        click.echo(
+            f'To create a branch, "{dataset_client.name}" can not be in draft status', err=True
```

Comment:
Will it be better if changing the message like this:
`Branch cannot be created from a draft`?